src/lib/storage/table.hpp

```diff
@@ -81,10 +81,14 @@ class Table : private Noncopyable {
    * @defgroup Accessing and adding Chunks
    * @{
    */
-  // returns the number of chunks (cannot exceed ChunkID (uint32_t))
+  // Returns the number of chunks, or, more correctly, the ID of the last chunk plus one (see get_chunk)
+  // This cannot exceed ChunkID (uint32_t).
   ChunkID chunk_count() const;
 
-  // returns the chunk with the given id
+  // Returns the chunk with the given id. If a previously existing chunk has been physically deleted by the
+  // MvccDeletePlugin, this returns nullptr. In the execution engine, it is the GetTable operator's job to
+  // filter these nullptrs and return only existing chunks to the following operator. Thus, all operators
+  // should assert the absence of nullptrs.
```

Comment:
"should assert that this function always returns a chunk."?

Reply:
Thus, all other operators should not accept nullptrs and instead assert that this function returned a chunk

?

Reply:
Accept > expect?

Reply:
Not really what I wanted to say. If you wanted to use expect, I would expect "should not have to expect".